npm_tasks/gh-pages-win.bat

```diff
@@ -0,0 +1,21 @@
+
+PROJECT_ROOT="%(git rev-parse --show-toplevel)"
+CURRENT_BRANCH="%(git rev-parse --abbrev-ref HEAD)"
+
+cd %PROJECT_ROOT
+
+git stash
+
+git checkout gh-pages
+git checkout %CURRENT_BRANCH -- dist/*
+git checkout %CURRENT_BRANCH -- public/*
+
+git add dist/*
+git add public/*
+
+git commit -m "deployed from %CURRENT_BRANCH"
+git push --force origin gh-pages
```

Comment:
Here is using force push, which might not be appropriate, I am still looking for answer, kindly comment ~